.template/addons/github/.github/workflows/publish_wiki.yml.tt

```diff
@@ -2,17 +2,35 @@ name: Publish Wiki
 
 on:
   push:
-    paths:
-      - .github/wiki/**
     branches:
       - develop
+    paths:
+      - .github/wiki/**
+
+env:
+  USER_NAME: github-wiki-workflow
+  USER_EMAIL: bot@nimblehq.co
 
 jobs:
   publish:
-    name: Publish Wiki
-    uses: nimblehq/github-actions-workflows/.github/workflows/publish_wiki.yml@0.1.0
-    with:
-      USER_NAME: github-wiki-workflow
-      USER_EMAIL: ${{ secrets.GH_EMAIL }}
-    secrets:
-      USER_TOKEN: ${{ secrets.GH_TOKEN }}
+    name: Publish Github Wiki
+    runs-on: ubuntu-latest
+    timeout-minutes: 1
+
+    steps:
+      - name: Cancel previous runs
+        uses: styfle/cancel-workflow-action@0.8.0
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
```

Comment:
Do we need this block? Isn't specifying the `branch` and `path` enough?

Reply:
The workflow doesn't work as expected if this block is removed. The job runs successfully, but changes in wiki directory doesn't get reflected on github wiki.

Reply:
I see,  Thanks for explaining!
let's use the newer version then `actions/checkout@v3`!